### PR TITLE
83 improve description styling

### LINKED ISSUE #83: Improve styling for description
__Is your feature request related to a problem? Please describe__

Improve styling for description.
* Currently description texts are hard to distinguish from labels
* See https://github.com/bpmn-io/bpmn-properties-panel/pull/16

__Describe the solution you'd like__

Improve styling via css as described [in this comment](https://github.com/bpmn-io/bpmn-properties-panel/pull/16#pullrequestreview-704991425)

__Describe alternatives you've considered__

Leave as-is

__Additional context__

### PR DESCRIPTION
**Pull request by MaxTru**
closes #83

### CODE REVIEW

**Comment by MaxTru** [comment]
Example one
Old:
Screenshot_20210713_143351


New:
Screenshot_20210713_143337

**Comment by MaxTru** [comment]
Example two

old:
Screenshot_20210713_143456

new:
Screenshot_20210713_143619

File: assets/properties-panel.css [hunk 1]
```diff
@@ -262,7 +261,12 @@
 }
 
 .bio-properties-panel-description {
-  margin-top: 7px;
+  display: block;
+  margin: 4px 0 8px;
+  color: #999;
```
Comment (pinussilvestrus) [inline assets/properties-panel.css:266]: Can we try to use [semantic color variables here](https://github.com/bpmn-io/properties-panel/blob/44d12e9c085030d935e628063554627596c71c90/assets/properties-panel.css#L14)?

So something like

```css
.bio-properties-panel {
  --color-999999: #999999;

  --color-description: var(--color-999999);
}

.bio-properties-panel-description {
  color: var(--color-description);
}
```

Comment (MaxTru) [reply assets/properties-panel.css]: :+1: good point

**Comment by nikku** [comment]
I like a lot.

**Review by pinussilvestrus — APPROVED** [review]
I like 👍 

For reference, a bigger example for the description (timers)

image

**Review by andreasgeier — APPROVED** [review]
✅ &nbsp; Nice job!

**Pull request merged**